woocommerce/navigation
# Update menu function args to an array

* Updates the menu item arguments to an array to improve readability and extensibility.
* Renames `slug` to `id` to reduce ambiguity.

### Testing

1. Make sure that tests are passing and no errors occur.
1. Make sure that menu output on wc/non-wc pages is correct.
